src/test/java/seedu/anilist/logic/commands/ConfirmClearCommandTest.java

```diff
@@ -22,7 +22,7 @@
 public class ConfirmClearCommandTest {
 
     /** {@code Predicate} that always evaluate to false */
-    private Predicate<Anime> predicateShowNoAnime = unused -> false;
+    private final Predicate<Anime> predicateShowNoAnime = unused -> false;
```

Comment:
final should be all caps

Reply:
YAH SO WE CANT USE ALL CAPS FOR CONSTANTS IN TESTS FOR SOME REASON